testlib/src/test/resources/formatter/properties/valid_xml_profiles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="12">
+	<profile kind="First" name="Used Profile" version="SomeVersion">
+		<setting id="valid_xml_profiles.xml.string" value="string" />
+		<setting id="valid_xml_profiles.xml.true" value="Gets overridden by next line" />
+		<setting id="valid_xml_profiles.xml.true" value="true" />
+		<setting id="valid_xml_profiles.xml.42" value="42" />
+		<setting id="valid_xml_profiles.xml.null"/>
+		<setting id="common" value="valid_xml_profiles.xml" />
+	</profile>
+	<profile kind="Second" name="Skipped Profile" version="AnotherVersion">
+		<setting id="valid_xml_profiles.xml.string" value="Second profile should be skipped" />
+		<setting id="valid_xml_profiles.xml.true" value="Second profile should be skipped" />
+		<setting id="valid_xml_profiles.xml.42" value="Second profile should be skipped" />
+		<setting id="valid_xml_profiles.xml.null" value="Second profile should be skipped" />
+		<setting id="common" value="Second profile should be skipped" />
+	</profile>
+</profiles>
```

Comment:
I mean throw either a RuntimeException or an Error.  People ignore warnings.

What is the use case where a user will have a file with two profiles, and their intention is to silently ignore the second profile?

What is the value of supporting that use case, compared to the error mode that the user thinks they have saved their profile, but actually they have saved it only to the second profile?

My strong opinion is that we should fail hard when a file has multiple profiles. What is the use case where this behavior is against the user's wishes?